modules/local/process/bowtie_align.nf

```diff
@@ -0,0 +1,46 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process BOWTIE_ALIGN {
+    tag "$meta.id"
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+
+    // Note: 2.7X indices incompatible with AWS iGenomes.
+    conda (params.enable_conda ? "bioconda::bowtie=1.2.3" : null)
```

Comment:
I just saw that bowtie 1.3.0 is out. Since we are already at it, should we update to it?

Reply:
Yeah, good point. Let's update it.
I will do some local testing as well on monday to get the tests running, and hopefully make some final commits to this PR then.